docs/SOPs/update_ena_runs_SOP.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: default
+title: Update ENA runs SOP
+parent: SOPs
+---
+## Table of contents
+
+{: .no_toc .text-delta }
+
+1. TOC {:toc}
+## Overview
+This is the SOP for fixing datasets in the issue: ebi-ait/hca-ebi-wrangler-central#250
+
+## Requirements
+1. Gain access to the EBI cluster
+   ```
+   ssh ebi-cli.ebi.ac.uk
+   ```
+2. Install `gsutil` in your environment in the EBI cluster and log in using your EMBL-EBI google account.
+   You could follow instructions from https://cloud.google.com/storage/docs/gsutil_install to install `gs_util`. 
+   See more details [here](https://ebi-ait.github.io/hca-ebi-dev-team/admin_setup/Setting-up-access-to-Terra-staging-area.html#using-your-google-account) about setting up your access to Terra staging area.
+3. Get DSP's Webin [credentials](https://console.aws.amazon.com/secretsmanager/home?region=us-east-1#!/secret?name=ingest%2Fwebin-creds). (Only an ingest developer has access to this atm)
+4. Clone the ingest-archiver repository. The scripts that will be used is in `ena` directory of that repo.
+   ```
+   git clone https://github.com/ebi-ait/ingest-archiver.git
+   pip install -r requirements.txt
+   ```
+5. Get your JWT Token from Ingest UI.
+   1. Log in in Ingest UI https://contribute.data.humancellatlas.org/login using the account which has the WRANGLER role.
+   1. In Chrome, right click and select _Inspect_ to open the developer console. Select the Network tab.
+   1. Refresh the page, `https://contribute.data.humancellatlas.org/home`
+   1. Check the _Authorization_ in headers of the request to `https://api.ingest.archive.data.humancellatlas.org/auth/account`
+   1. Copy the token after _Bearer_ prefix: 
+      ```
+      Authorization: Bearer <copy the very long string of random characters>
+      ```
+   1. That token has 1 hr validity. The token will be needed in the submitter script later.
+## Steps
+
+1. Get the list of sequencing runs to be suppressed. This can be downloaded as TSV/JSON from the ENA Browser. 
+   
+2. Make sure
+   that the metadata in Ingest contains sequencing experiment accessions. The submitter script will raise an error if
+   any of the assay processes has no accession. The assay processes in the submission should have the following
+   property:
+
+    ```
+    "insdc_experiment": {
+      "insdc_experiment_accession": "ERX4319109"
+    }
+    ```
+
+2. Clear the sequencing run accessions in file metadata. The following should not be in the file metadata json:
+    ```
+    "insdc_run_accessions": [
+      "ERR6449905"
+    ]
+    ```
+   
+   Update script to have a jwt token from the Ingest UI then run the following: 
```

Comment:
seems to me that the script is `ena/clear_run_accession_from_files.py` (https://github.com/ebi-ait/ingest-archiver/blob/task/submit-10x-fastq-files-integration-tests/ena/clear_run_accession_from_files.py). I'm making changes accordingly and leaving this comment here in case this isn't the case